### DISCUSSION TITLE
Starting a command in a new / specifc tab

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [zellij-org/zellij#3671] [Q&A]
I was able to do this using the following sequence of actions:

```sh
zellij action go-to-tab-name --create "my-tab-name" \
&& zellij action new-pane -- <command> \
&& zellij action focus-previous-pane \
&& zellij action close-pane
```

Is this the only way right now?

I tried using `--in-place`, but that replaced the pane I've initiated the sequence from, and not the destination pane.

**Comment by jeffnm** [comment]
I'm having the same challenge. There also doesn't seem to be an action for breaking a pane, so I can't run the command in a pane and then break it into a tab. Your sequence got me to something that works, even if it isn't ideal, so thanks for posting it. 

My use case is to open an editor in a new tab from yazi, and a sequence like this is getting me there. 
```bash
zellij action go-to-tab-name --create $(basename $1) \
&& zellij edit $0 \
&& zellij action focus-previous-pane \
&& zellij action close-pane
